tests/vote_test.coffee

```diff
@@ -14,3 +14,18 @@ describe 'user voting', ->
     it 'should tell the user how to vote', ->
       room.user.say 'alice', '@hubot vote?'
       expect(room.messages[1][1].split("\n")[0]).to.eql "Voting allows you to pretend you have the power of a god."
+
+  context 'custom timeout', ->
+    beforeEach ->
+      this.clock = sinon.useFakeTimers()
+
+    afterEach ->
+      this.clock.restore()
```

Comment:
`@` is an alias for `this`, it can be written `@clock`, but I didn't even notice in this case since I never saw a `@clock =`